package.json

```diff
@@ -52,7 +52,9 @@
     "storybook:build": "panda codegen && storybook build",
     "changeset": "changeset",
     "release": "yarn build && changeset publish",
-    "prepare": "husky install"
+    "postinstall": "husky install",
+    "prepack": "pinst --disable",
+    "postpack": "pinst --enable"
```

Comment:
Nice! Appreciate you updating this! Verified that `postinstall` script did run appropriately after installing deps locally.